api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessagePayloadFactory.java

```diff
@@ -61,6 +58,12 @@ public static EmailMessagePayload buildEmailMessagePayload(PaymentCycle paymentC
                 .build();
     }
 
+    public static ReportClaimMessagePayload buildReportClaimMessagePayload(Claim claim) {
+        return ReportClaimMessagePayload.builder()
+                .claimId(claim.getId())
```

Comment:
I think we'll also need to store the state of the claim (new/updated/rejected) as we can't assume the status of the claim will accurately reflect it later.

Reply:
As discussed, we can add that in when we come to implement the story that needs it.

Reply:
As discussed, this will be added with the additional fields that are required.